controllers/imagepolicy_controller.go

```diff
@@ -148,6 +148,20 @@ func (r *ImagePolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	var err error
```

Comment:
This declaration could still do with removing, it's redundant. I think the shadowing only escapes the linter because policer is a new variable.
I'd also like to avoid the reuse of `err` afterward. It made some sense before (it might be the error returned by `PolicerFromSpec`), but now it's confusing.

Reply:
I've just move this declaration just above the block that fetches the latest tag.

Reply:
I'll just move this error before the block that fetches the latest tag with a different name.

Reply:
Changed my mind, just removing the first declaration :sweat_smile: